torch/_inductor/decomposition.py

```diff
@@ -471,6 +471,29 @@ def randn_like(self, *, dtype=None, device=None, **kwargs):
     )
 
 
+@register_decomposition(aten.empty_like)
```

Comment:
I guess the key concern here is that `empty_like` preserves strides while it is not with the decomposition implementation below.

Reply:
Inductor's version of `empty_like` doesn't preserve strides.  It won't get realized and will just be inlined into the users.

Reply:
It is a little different because the decomps run at aot_autograd tracing time, so the recorded traces wont have correct strides. inductor graph doesnt preserve strides but does rely on the traced graph for stride correctness